pennylane/tape/tape.py

```diff
@@ -1002,23 +1003,23 @@ def get_resources(self):
 
         >>> tape.get_resources()
         {'Hadamard': 2, 'RZ': 1, 'CNOT': 2, 'Rot': 1}
+
         """
-        if self._resources is None:
-            self._resources = {}
 
-            for op in self.operations:
-                if op.name not in self._resources.keys():
-                    self._resources[op.name] = 1
-                else:
-                    self._resources[op.name] += 1
+        warnings.warn(
+            "``tape.get_resources`` will be deprecated after v0.16 "
```

Comment:
Do we want to say when it will be removed?
E.g.
"``tape.get_resources`` is now deprecated and will be removed in v0.17"